### ISSUE TITLE
False positive in NoLineWrap Check for static imports

### ISSUE THREAD

**Issue by kukreja-vikramaditya** [checkstyle/checkstyle#3892]
```
$ cat config.xml 
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
<module name="Checker">
    <property name="charset" value="UTF-8"/>
        <module name="TreeWalker">
             <module name="NoLineWrap">
        </module>
    </module>
</module>


$ cat Main.java 
import
static
   java
   .math
   .BigInteger
   .ONE;                 //Should show violation for IMPORT

import java.math.BigInteger;

public final class Main {
  public static void main(String... args) {
    BigInteger max = ONE;
  }
}

$ java -jar checkstyle-7.6-all.jar -c config.xml Main.java
Starting audit...
Audit done.
```
---

I expect an error during audit process like the google guidelines mentioned [here](https://google.github.io/styleguide/javaguide.html#s3.3.2-import-line-wrapping). But the [NoLineWrap](http://checkstyle.sourceforge.net/config_whitespace.html#NoLineWrap) is unable to figure this. If `static` keyword is removed, audit prints the expected result.

**Comment by MEZk** [comment]
@kukreja-vikramaditya 
Please, add Checkstyle configuration xml into issue description.

**Comment by rnveach** [comment]
@romani The issue looks valid to me.
http://checkstyle.sourceforge.net/config_whitespace.html#NoLineWrap
We already have `IMPORT` token, and we by default say we will verify `import` statements.
It seems we are just missing the `STATIC_IMPORT` token in acceptable and default.

**Comment by rnveach** [comment]
Fix is merged
